### PR TITLE
LPS-150510 Test Autom. for Bug: Basic document cannot be selected in the subtype filter

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
Motivation
=======
[LPS-150510](https://issues.liferay.com/browse/LPS-150510): Test Automation for LPS-136346:
↳[LPS-136346](https://issues.liferay.com/browse/LPS-136346): Basic document cannot be selected in the subtype filter

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
ci:test:tango

**Comment by Luizcarlosqueiroz** [comment]
@CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -3018,6 +3018,32 @@ definition {
 		}
 	}
 
+	@description = "LPS-150510 Bug: Basic document cannot be selected in the subtype filter"
+	@priority = "4"
+	test FilterBySubtypeBasicDocument {
+		task ("Create a Basic Document") {
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:3024]: As this is a priority 4 test, it should be included in the tango-acceptance routine. Please add:
 _property portal.acceptance = "true";_

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by Luizcarlosqueiroz** [comment]
Hey @CarlosBrichete ! Property added. Should we run the ci tests again?

**Comment by CarlosBrichete** [comment]
> Hey @CarlosBrichete ! Property added. Should we run the ci tests again?

No, there's no need :)

**Comment by CarlosBrichete** [comment]
ci:forward

**Review by CarlosBrichete — APPROVED** [review]

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**